<?xml version="1.0" ?>
<launch>
  
  <arg name="capsicum_cloud_topic" default="/capsicum/points" />
  <arg name="capsicum_model_topic" default="/capsicum/model/points" />
    
  <node name="superquadric_fitter" pkg="superquadric_fitter" type="superquadric_fitter" output="screen">
  
  <param name="capsicum_cloud_topic" value="$(arg capsicum_cloud_topic)" />
  <param name="capsicum_model_topic" value="$(arg capsicum_model_topic)" />

  </node>

</launch>

